packages/react-components/src/components/MessageThread.tsx

```diff
@@ -517,6 +518,16 @@ export type MessageThreadProps = {
    * @beta
    */
   onRenderFileDownloads?: (userId: string, message: ChatMessage) => JSX.Element;
+  /**
+   * Optional file download error message to render.
+   * @beta
+   */
+  fileDownloadErrorMessage?: string;
+  /**
+   * Optional callback to dismiss the file download error.
```

Comment:
Misleading comment. 
This callback runs when someone dismisses an error bar.

Comment should be something like
`Optional callback. Runs whenever a download error message is dismissed`